samwise.js

```diff
@@ -12,12 +12,13 @@
     var autoplayDefault = 0;
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.